dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -236,8 +236,12 @@ def navigate_to(url)
   wait_until {@browser.execute_script("return $(\"#{selector}\").is(':checked');") == negation.nil?}
 end
 
+def jquery_is_defined
+  "(typeof jQuery !== 'undefined') && ($ !== 'undefined')"
+end
+
 def jquery_is_element_visible(selector)
-  "return $(#{selector.dump}).is(':visible') && $(#{selector.dump}).css('visibility') !== 'hidden';"
+  "return #{jquery_is_defined} && $(#{selector.dump}).is(':visible') && $(#{selector.dump}).css('visibility') !== 'hidden';"
```

Comment:
I don't think we need to test that jquery is defined when verifying that an element is visible. It will cause this to return false.

Instead, the act of loading a page should wait for jquery to become available. So I think we can just remove this check here and the rest of it is good.

Reply:
The exception is thrown after step #2 (detailed in PR description), where a jQuery seems to have been executed successfully, but some page reload seems to have happened since leading to the exception. I think adding the additional check for $ increases our resilience in general, but don't think it would address this specific behavior we are seeing.

Reply:
These tests are also executed as part of drone and seem to work working successfully. So, the overall condition is working as expected.